examples/player/simid_player.js

```diff
@@ -1,16 +1,16 @@
 const NO_REQUESTED_DURATION = 0;
 const UNLIMITED_DURATION = -2;
 
-/**
- * All the logic for a simple SIMID player
+/*
+  All the logic for a simple SIMID player
  */
```

Comment:
revert this

Reply:
i did not make this change purposefully, reverted